kernel/src/lib.rs

```diff
@@ -121,7 +121,7 @@ pub struct MemoryBlock {
 pub fn num_pages(nbytes: u64) -> usize {
     let npages = nbytes / PAGE_SIZE as u64;
     let remainder = nbytes % PAGE_SIZE as u64;
-    if remainder != 0 || nbytes < PAGE_SIZE as u64 {
```

Comment:
We no longer need to check if `nbytes < PAGE_SIZE` ?

Reply:
Yeah, @chrisdickinson pointed out in another comment that it's actually alright: https://github.com/extism/extism/pull/472#discussion_r1330777361